pkg/cmd/cli/cmd/deploy.go

```diff
@@ -350,7 +350,7 @@ func (o DeployOptions) cancel(config *deployapi.DeploymentConfig) error {
 				continue
 			}
 
-			deployment.Annotations[deployapi.DeploymentCancelledAnnotation] = deployapi.DeploymentCancelledAnnotationValue
+			deployment.Annotations[deployapi.DeploymentCancelledAnnotation] = unversioned.Now().Format(time.RFC3339)
```

Comment:
> will this work with different timezones?

Good question. unversioned.Now is using time.Now under the hood which captures the local time. We could potentially use time.UTC for storing the time and always restore with Local() but that's probably not a good idea. Now, I am wondering if `oc logs --since` has the same issue. cc: @smarterclayton